wp-content/plugins/cp-gp-cli/inc/class-language-pack.php

```diff
@@ -230,7 +230,7 @@ private function build_language_packs( $data ) {
 			// TODO install the real `zip` and use it instead
 			$result = $this->execute_command(
 				sprintf(
-					'zip-hack -9 -j %s %s %s 2>&1',
+					'zip-hack %s %s %s 2>&1',
```

Comment:
I fixed this in the `zip-hack` script.  Let's leave it in the code here, otherwise we will forget to put it back when we use the real `zip` again.

Reply:
Updated in e1d70e7